app/models/manageiq/providers/workflows/automation_manager/authentication.rb

```diff
@@ -0,0 +1,3 @@
+class ManageIQ::Providers::Workflows::AutomationManager::Authentication < ManageIQ::Providers::AutomationManager::Authentication
+  validates :name, :presence => true, :uniqueness_when_changed => {:scope => [:tenant_id]}
```

Comment:
TODO enforce a format that doesn't allow for spaces/periods so the JSON path regex will work, thinking non-fqdn hostname pattern would be good (something like `[a-zA-Z0-9\-]`)

Reply:
:+1: This also will prevent json injection attacks